Library/Homebrew/mktemp.rb

```diff
@@ -40,7 +46,15 @@ def to_s
   end
 
   def run
-    @tmpdir = Pathname.new(Dir.mktmpdir("#{@prefix.tr "@", "AT"}-", HOMEBREW_TEMP))
+    prefix_name = @prefix.tr "@", "AT"
+    if @retain_in_cache
```

Comment:
```suggestion
    if retain_in_cache?
```
Or, if we're going to access the instance variable directly, drop the `:retain_in_cache?` method.

Reply:
Done